go/prometheusexporter/collector/collector.go

```diff
@@ -981,9 +983,9 @@ func (c *Collector) ProcessMessage(routingKey string, data map[string]interface{
 		m = c.processLogMessage(routingKey, data)
 	case strings.HasPrefix(routingKey, pageRoutingKey):
 		m = c.processPageMessage(routingKey, data)
```

Comment:
I think page messages suffer from the same problem (essentially every metric coming from logjam-fhttpd will likely have that "issue").
Mobile metrics as well.